v4/weaviate/weaviate_client.go

```diff
@@ -59,6 +61,11 @@ type Client struct {
 // All these models are provided in the sub module "github.com/semi-technologies/weaviate/entities/models"
 func New(config Config) *Client {
 	con := connection.NewConnection(config.Scheme, config.Host, config.ConnectionClient, config.Headers)
+	if time.Now().UTC().After(time.Date(2023, 1, 19, 0, 0, 0, 0, time.UTC)) {
+		fmt.Printf("On 2023-01-19 the repositoy of the weaviate-go-client moved from https://github.com/semi-technologies/weaviate-go-client to" +
+			"https://github.com/weaviate/weaviate-go-client . Please update your imports." +
+			"This warning will disappear after updating.")
```

Comment:
Add new line and the very end of the text (and change to `fmt.Print`) or just use `fmt.Println`

Reply:
thx, done